### DISCUSSION TITLE
Is there a way to lock / disable any settings set by helm within the UI?

### DISCUSSION THREAD

**Discussion by nogweii** [longhorn/longhorn#7922] [Q&A]
This is mostly to keep things in-sync. If I set a value through the [`defaultSettings` section of the helm chart](https://github.com/longhorn/charts/tree/v1.6.x/charts/longhorn#system-default-settings), is there a way to ensure Longhorn will always use that value?

I'm worried about past me editing one of those settings, and then future me being confused as to how that value "got out of sync" from my repo of YAML manifests.

**Comment by derekbit** [comment]
Longhorn will use the values you edited.

**Reply by nogweii** [reply]
Sorry, I don't follow. I'm looking to set the values through `defaultSettings` and preventing them from being edited. To ensure that only the value from `defaultSettings` is respected, guaranteeing the values in my GitOps repo are what's used.

**Reply by derekbit** [reply]
Longhorn doesn't support locking the settings for now.
Can you elaborate more on the use case? Will others edit the settings in your environment?
